ocaml/database/db_backend.ml

```diff
@@ -67,7 +67,7 @@ let blow_away_non_persistent_fields (schema: Schema.t) db =
     db
 
 let db_registration_mutex = Mutex.create ()
-let foreign_databases: ((API.ref_session, Db_ref.t) Hashtbl.t) = Hashtbl.create 5
+let foreign_databases: ((string, Db_ref.t) Hashtbl.t) = Hashtbl.create 5
```

Comment:
It doesn't actually represent anything to the db layer - it's an opaque string that the caller has to use to identify the db.